data/pop-upgrade.sh

```diff
@@ -57,10 +57,10 @@ dpkg_configure () {
     # Validate the exit status
     dpkg --configure -a
 }
-
 apt_install_fix () {
     message -i "Checking for package fixes"
     env LANG=C apt-get -o Dpkg::Options::="--force-overwrite" \
+        -o Dpkg::Options::="--force-all" \
```

Comment:
If you use force-all, you probably don't need the other two force options

Reply:
May as well apply it to all other install commands, too.

Reply:
I don't think forcing all the dpkg force options is safe, though. If it is possible to find out which force option specifically fixes ppp upgrade then that would be better

Reply:
Reviewing the options, it must be one of these:

```
force-breaks
force-conflicts
force-depends
force-depends-version
```